package.json

```diff
@@ -11,7 +11,8 @@
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
     "react-headroom": "^2.1.2",
-    "react-helmet": "^3.1.0"
+    "react-helmet": "^3.1.0",
+    "react-router": "^4.0.0"
```

Comment:
Can you remove `"react-router": "^4.0.0"` and then delete your `node_modules` folder and then run `yarn add react-router@2.6.1` or `npm i --save react-router@2.6.1`